### PR TITLE
Pass in kwargs from PostgreSQLClient to ConnectionPool

### PR DESCRIPTION
**Pull request by aaronfriedman6**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.0.8 - 3/3/23
+- Pass in all kwargs from PostgreSQLClient to ConnectionPool so that all ConnectionPool settings
+can be set from the
```
Comment (mwbenowitz) [inline CHANGELOG.md:5]: Looks like there should be more here!

File: tests/test_postgresql_client.py [hunk 2]
```diff
@@ -24,13 +24,14 @@ def test_init(self, test_instance):
     def test_init_with_kwargs(self):
         test_instance = PostgreSQLClient(
             'test_host', 'test_port', 'test_db_name', 'test_user',
-            'test_password', min_size=5, max_size=10)
+            'test_password', min_size=5, max_size=10, max_idle=15.0)
```
Comment (mwbenowitz) [inline tests/test_postgresql_client.py:27]: Don't see a reference to `max_idle` anywhere else, maybe a docstring for the class could explain what common kwargs we expect to be passed in? Unless this is just to show that it accurately picks up random kwargs and sets them correctly.

**Review by mwbenowitz — APPROVED** [review]
Looks solid, just 2 questions that shouldn't block merging

File: tests/test_postgresql_client.py (hunk 2 above)
Comment (aaronfriedman6) [reply tests/test_postgresql_client.py:27]: At the moment this is mostly the latter -- just testing that kwargs is passed in. Depending on how the connection leakage issue shakes out, `max_idle` may end up becoming important, but for now it doesn't need special documentation I don't think.

File: CHANGELOG.md (hunk 1 above)
Comment (aaronfriedman6) [reply CHANGELOG.md:5]: Whoops!

**Review by mwbenowitz — APPROVED** [review]

**Pull request merged**